python/huggingfaceserver/huggingfaceserver/vllm/vllm_model.py

```diff
@@ -61,10 +62,8 @@ def load(self) -> bool:
         return self.ready
 
     async def healthy(self) -> bool:
-        try:
-            await self.vllm_engine.check_health()
-        except Exception as e:
-            raise ModelNotReady(self.name) from e
+        # check_health() may throw exceptions which are caught in OpenAIEndpoints class
```

Comment:
Throwing an exception here explicitly feels weird. Moved the exception handling to endpoints.py below.